### PR TITLE
Add tests and improvements

### PR DESCRIPTION
**Pull request by samuelallan72**
Add tests and some other niceties. Some may be controversial.

### CODE REVIEW

**Comment by samuelallan72** [comment]
@kennib There's now a travis config included - you'll need to activate travis for this repo first though.

File: server.py [hunk 1]
```diff
@@ -66,11 +65,13 @@ def db_fetch(key):
...
     return None
 
-def db_delete(key):
+def db_delete(key: str) -> bool:
   conn = get_db()
   c = conn.cursor()
   c.execute('DELETE FROM store WHERE key = ?', (key,))
+  did_delete = c.rowcount > 0
   conn.commit()
+  return did_delete
```
Comment (kennib) [inline server.py:74]: Can we make this a JSON object? e.g. `{"key": "whatever", "deleted": true}`

**Review by kennib — CHANGES_REQUESTED** [review]
Looks good. Just one small change

File: server.py (hunk 1 above)
Comment (samuelallan72) [reply server.py:74]: done :)

**Pull request merged**